dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/SystemSettingController.java

```diff
@@ -246,7 +246,14 @@ private Serializable getSystemSettingOrTranslation( String key, String locale )
                 }
             }
 
-            return systemSettingManager.getSystemSetting( settingKey.get() );
+            Serializable systemSetting = systemSettingManager.getSystemSetting( settingKey.get() );
+
```

Comment:
Could be simplified with:

`ObjectUtils.firstNonNull( systemSetting, StringUtils.EMPTY );`